### DISCUSSION TITLE
Hi everyone!

### DISCUSSION THREAD

**Discussion by chenzejun** [Schweinepriester/github-profile-achievements#902] [General]
have you got the achievements?

**Comment by nithinlycanz** [comment]
heyy Do you have any questions??

**Comment by Saqib198** [comment]
Not Yet...
Any one Help me in it...

**Comment by codingCapricorn** [comment]
@chenzejun @nithinlycanz @Saqib198                                                                                                                                                              For getting achievement : https://github.com/Schweinepriester/github-profile-achievements

**Comment by 17351269** [comment]
Thank

در تاریخ چهارشنبه 29 مه 2024، 21:59 Coding Capricorn <
***@***.***> نوشت:
